application/views/themeOptions/index.php

```diff
@@ -58,19 +60,50 @@
                             </thead>
 
                             <tbody>
-                            <?php foreach ($oSurveyTheme->templatesWithNoDb as $oTemplate) : ?>
-                                <?php /** @var TemplateManifest $oTemplate */ ?>
+                            <?php /** @var TemplateManifest $oTemplate */ ?>
+                            <?php $surveyThemeIterator = 0 ?>
+                            <?php foreach ($templatewithNoDb as $key => $oTemplate) : ?>
                                 <tr class="odd">
                                     <td class="col-lg-1"><?php echo $oTemplate->getPreview(); ?></td>
                                     <td class="col-lg-2"><?php echo $oTemplate->sTemplateName; ?></td>
                                     <td class="col-lg-3"><?php echo $oTemplate->getDescription(); ?></td>
                                     <td class="col-lg-2"><?php eT('XML themes'); ?></td>
                                     <td class="col-lg-1"><?php echo $oTemplate->config->metadata->extends; ?></td>
-                                    <td class="col-lg-2"><?php echo $oTemplate->getButtons(); ?></td>
+                                    <?php if (TemplateConfig::isCompatible($oTemplate->path . 'config.xml')): ?>
+                                        <td class="col-lg-2"><?php echo $oTemplate->getButtons(); ?></td>
+                                    <?php else: ?>
```

Comment:
I think it's better to look at this HTML than to just read it. :) But Tibor/I would need instructions on how to do that.

Reply:
this is just displayinga button and modal instead of the normal buttons, should be tested in manual testing yeah